File: Python/algoritmo004/main.py
def executar():
    print("Vejo a previsão do tempo")

    fazendo_sol = input(
        "Está fazendo sol? (s/n): "
    )

    if fazendo_sol.lower() == "s":
        print("Vou à praia")
    else: 
        print("Vou estudar")

    print("Almoçar")
    print("Ver televisão")
    print("dormir")

executar()

File: Java/algoritmo004/Main.java
package algoritmo004;

import java.util.Scanner;

public class Main {
    public static void executar(){
        Scanner scanner = new Scanner(System.in);

        System.out.println("Vejo a previsão do tempo");
        System.out.print("Está fazendo sol? (s/n): ");
        String fazSol = scanner.nextLine();

        if (fazSol.equalsIgnoreCase("s")){
            System.out.println("Vou à praia");
        } else {
            System.out.println("Vou estudar");
        }

        System.out.println("Almoçar");
        System.out.println("Ver televisão");
        System.out.println("Dormir");
    }

    public static void main (String[] args){
        executar();
    }
}